doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -269,4 +296,50 @@ You are now all set to start the User OS (UOS)
 
 **Congratulations**, you are now watching the User OS booting up!
 
-.. _Clear Linux: https://clearlinux.org
+
+
+Enabling network sharing 
+************************
+
+after booting up SOS and UOS, you may need to run some basic test and install addational pakcages
+enabling network in UOS is must, enabling the requiring tap and networking bridge in the SOS 
+to enable networking in the guest OS is important, you can create a script file to setup.
+
+below is an example for you reference:(verifed in Ubuntu 14.04 and 16.04 as the SOS) 
```

Comment:
Hold on here.  We were using Clear Linux as the Service and User OS, and now you're talking about Ubuntu)?  Was this verified with Clear Linux, if so, say that instead.

Reply:
no, this guide is for ubuntu as SOS and Clear linux as UOS only,  for Clear Linux as the SOS, the script to setup tap is different